src/csharp/experimental/build_unitypackage.sh

```diff
@@ -1,32 +0,0 @@
-#!/bin/sh
```

Comment:
I assume this bash script was just unused?

Reply:
Yes.